src/app/clinic/clinic_facade.cpp

```diff
@@ -30,35 +30,45 @@ void Clinic::appendCalendar(const std::shared_ptr<Calendar>& calendar)
     calendar_ = calendar;
 }
 
-std::vector<std::shared_ptr<Doctor>>& Clinic::getDoctors()
+void Clinic::appendWarehouse(const std::shared_ptr<Warehouse>& warehouse)
+{
+    warehouse_ = warehouse;
+}
```

Comment:
I think that we should change name to setWarehouse as "append" is misleading.

I mean, we have just 1 Warhouse which we set every time the function is called.
We do not add another Warehouse each time, as the name suggests.